clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -1467,4 +1476,51 @@ int updateAndGet(int actualSize, boolean update) {
             }
         }
     }
+
+    /**
+     * Refer KAFKA-16305.
```

Comment:
Can we summarize the description from the ticket in a couple of sentences instead of adding a reference to the ticket?

Reply:
Thanks! Addressed in [d954393](https://github.com/apache/kafka/pull/15434/commits/d9543932dcb63efb45c2d6e5704f044cb44ba95a)